Create ui-c8y-1023-69-0-added-decimal-place-configuration-for-computed-properties-11794-graft-release-cd.md (#4566)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-69-0-added-decimal-place-configuration-for-computed-properties-11794-graft-release-cd.md

---
date: ""

> title: Added possibility to configure decimal places for computed properties.

Reviewer: ```suggestion
title: Decimal places can now be configured for computed properties in the asset selector
```

> title: Decimal places can now be configured for computed properties in the asset selector

Reviewer: ```suggestion
title: Decimal places can now be configured for computed properties
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66622
version: 1023.69.0
---

> You can now configure the number of decimal places shown for computed properties in the asset selector when selecting datapoints, allowing you to control the display precision to match your specific requirements. This feature is currently utilized in the new asset table v2 which can be enabled via the preview features page.

Reviewer: @janhommes @rahulc8y
"new asset table v2":
Is this the official name as shown in the preview features page?
Should the label for the type be "Preview" in such a case?

> You can now configure the number of decimal places shown for computed properties in the asset selector when selecting datapoints, allowing you to control the display precision to match your specific requirements. This feature is currently utilized in the new asset table v2 which can be enabled via the preview features page.

Reviewer: ```suggestion
You can now configure the number of decimal places shown for computed properties in the asset selector when selecting datapoints, allowing you to control the display precision to match your specific requirements. This behavior is currently utilized in the new **Asset table v2** feature, which can be enabled via the **Preview features** page.
```

> You can now configure the number of decimal places shown for computed properties in the asset selector when selecting datapoints, allowing you to control the display precision to match your specific requirements. This feature is currently utilized in the new asset table v2 which can be enabled via the preview features drawer.

Reviewer: @janhommes @rahulc8y 
"new asset table v2": 
Is this the official name as shown in the preview features page? 
Should the label for the type be "Preview" in such a case?

Reviewer 2: "Asset table v2" is the name of the feature showed in the preview drawer.

> You can now configure the number of decimal places shown for computed properties in the asset selector when selecting datapoints, allowing you to control the display precision to match your specific requirements. This feature is currently utilized in the new asset table v2 which can be enabled via the preview features drawer.

Reviewer: ```suggestion
You can now configure the number of decimal places shown for computed properties in the asset selector when selecting datapoints, allowing you to control the display precision to match your specific requirements. This feature is currently utilized in the new asset table v2 which can be enabled via the preview features page.
```

> You can now configure the number of decimal places shown for computed properties in the asset selector when selecting datapoints, allowing you to control the display precision to match your specific requirements.  This feature is currently utilized in the new asset table v2 which can be enabled via the preview features drawer.

Reviewer: ```suggestion
You can now configure the number of decimal places shown for computed properties in the asset selector when selecting datapoints, allowing you to control the display precision to match your specific requirements. This feature is currently utilized in the new asset table v2 which can be enabled via the preview features drawer.
```

> You can now configure the number of decimal places shown for computed properties in the asset selector when selecting datapoints, allowing you to control the display precision to match your specific requirements. This behavior is currently utilized in the new **Asset table v2** feature, which can be enabled via the **Preview features** page.

Reviewer 3: ```suggestion
You can now configure the number of decimal places displayed for computed properties in the asset selector when selecting datapoints, allowing you to control the display precision to match your specific requirements. This enhancement is available in widgets that use this component, such as the **HTML widget** and the preview version of the **[Asset table widget](https://cumulocity.com/docs/change-logs/#ui-c8y-1023-49-0-add-configurable-data-grid-as-asset-table-widget)**.
```

---
date: ""

> title: Added decimal place configuration for computed properties. (#11794) [GRAFT][release/cd] (#11798)

Reviewer 2: ```suggestion
title: Added possibility to configure decimal places for computed properties.
```
